build/packer/ansible/agent-aws.yml

```diff
@@ -198,6 +203,14 @@
         - ubuntu:noble
         - rockylinux:8
         - oraclelinux:9
+        - public.ecr.aws/e7j3v3n0/rpmbuild:3
+        - public.ecr.aws/e7j3v3n0/rpmbuild:2
+
+    - name: Add an SSH key
+      authorized_key:
+        user: ec2-user
+        state: present
+        key: https://github.com/ademidoff.keys
```

Comment:
should be removed

Reply:
Why? It's analogous (or even a potential replacement of) to [this](https://github.com/Percona-Lab/jenkins-pipelines/blob/master/vars/getSHHKeysPMM.groovy#L5-L9), except that in this case we don't bang `github.com` with too many requests for every instance.

Reply:
I believe it should stay in jenkins pipelines for better transparency.

Reply:
It's about efficiency, transparency is the same, I believe - both are public repos.

Reply:
One more call to github isn't that critical for efficiency.

it's not a common case to add ssh key to base image.
There we can clearly see in logs that these keys were added and we can control on which jobs to add these keys.
It also might be missed by security team if user leaves a team.

Reply:
I don't think your argument is valid about the security team - they don't monitor the pipelines. 

But OK, it's not a big deal - I'll remove it.

Reply:
One more reason to do it - we lost the ability to use the native functionality that AWS CLI provides to pass ssh public keys to EC2 instances when we migrated from Amazon Linux 2 to an Oracle 9 based OS. 

Before, it was possible with `aws ec2-instance-connect send-ssh-public-key`. It's no longer the case. Therefore, to troubleshoot an agent, there is currently no way to ssh to it. 

@talhabinrizwan has more context, we discussed it last week.

Reply:
Nevermind, it's removed.